src/morale_types.cpp

```diff
@@ -73,6 +73,8 @@ const morale_type &morale_type_data::convert_legacy( int lmt )
 
             morale_type( "morale_perm_filthy" ),
 
+            morale_type( "morale_perm_debug" ),
```

Comment:
Adding this here is quite pointless, there can't be any old savegame that uses this morale type.